src/Network/Network/ExpressRouteCircuit/Peering/Connection/SetAzureExpressRouteCircuitConnectionConfigCommand.cs

```diff
@@ -0,0 +1,104 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Network.Models;
+using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
+using System;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Network {
+    [CmdletOutputBreakingChange(typeof(PSExpressRouteCircuit), DeprecatedOutputProperties = new[] { "AllowGlobalReach" })]
```

Comment:
I'm not sure why you want to add a breaking change attribute to a new cmdlet?
If `AllowGlobalReach` is going to be deprecated, why not just do it now?

Reply:
AllowGlobalReach actually touches a lot many places and other cmdlets as well (Get, List, Remove etc).
The changes to actually deprecate it would take a much bigger effort (across teams). 
Current PR is to make an incremental enhancement to another aspect of the Global Reach. 
To be consistent, we can provide the alert message with the newly added Cmdlet as well. 

If that doesn't sound appropriate, please let me know, I can consult with the team and get back.